src/wifipaf/WiFiPAFConfig.h

```diff
@@ -97,8 +100,8 @@
 #define PAF_MAX_RECEIVE_WINDOW_SIZE 6
 #endif
 
-#if (PAF_MAX_RECEIVE_WINDOW_SIZE < 3)
-#error "PAF_MAX_RECEIVE_WINDOW_SIZE must be greater than 2 for PAF transport protocol stability."
+#if (PAF_MAX_RECEIVE_WINDOW_SIZE < (WIFIPAF_CONFIG_IMMEDIATE_ACK_WINDOW_THRESHOLD + 1))
+#error "PAF_MAX_RECEIVE_WINDOW_SIZE must be greater or equal than 2."
```

Comment:
The if condition does not say "2" though ... should we change the message to not be a fixed size? It seems it "has to be larger than the immediate ack window threshold". Then again this would just re-state the error if code and comments that re-state code are bad (example here: they end up being wrong as code is updated). Instead maybe it would be better to just say "mismatch" and explain in the comment why these relate to each other like they do.

Reply:
I will change "2" to WIFIPAF_CONFIG_IMMEDIATE_ACK_WINDOW_THRESHOLD and add the chapter in spec. It may be better to reference the spec for the detailed information.